### ISSUE TITLE
Univariate qplot() fails

### ISSUE THREAD

**Issue by cpsievert** [tidyverse/ggplot2#2494]
```r
qplot(data = mtcars, x = wt)
```

```r
Don't know how to automatically pick scale for object of type NULL. Defaulting to continuous.
`stat_bin()` using `bins = 30`. Pick better value with `binwidth`.
Error in pmin(y, 0) : object 'y' not found
```

**Comment by etiennebr** [comment]
I can replicate this with `tidyverse/ggplot2@3c9c504`. The CRAN version works fine.

**Comment by hadley** [comment]
Fixed in 79757ee84bb42f39044d409ce886708c8a910ca3
